### PR TITLE
uefi : fix broken build due to missing edk2 submodule

### PR DESCRIPTION
**Pull request by shjala**
This change updates EDK2 to stable202408.01. This is a minor update with only one additional commit compared the previous version which fixes missing subhook submodule issue by switching to TianoCore mirror.

https://github.com/tianocore/edk2/compare/edk2-stable202408...edk2-stable202408.01

For the curios, Adding a patch to fix the issue is not straightforward because the docker ADD  command clones the repository and initializes the submodules at once (before applying any patches).

### CODE REVIEW

File: pkg/uefi/Dockerfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ COPY rpi /rpi
 FROM build AS build-amd64-versions
 
 ENV EDK_VERSION edk2-stable202408
```
Comment (OhmSpectator) [inline pkg/uefi/Dockerfile:32]: `edk2-stable202408.01` ?

Comment (shjala) [reply pkg/uefi/Dockerfile:32]: just a comment above it instead of renaming?

Comment (shjala) [reply pkg/uefi/Dockerfile:32]: sure, I'm lazy.

**Review by OhmSpectator — APPROVED** [review]

**Comment by OhmSpectator** [comment]
Hm... In theory, the tests should have success, then... And they are still red =(

**Comment by shjala** [comment]
> Hm... In theory, the tests should have success, then... And they are still red =(

This has nothing to do with the EVE stucking at boot in Eden, this just fixes building EDK2 from source which is broken (even old versions are broken).

**Comment by OhmSpectator** [comment]
@shjala but we still have to update the hashes in the dependent Dockerfiles...

**Pull request merged**